src/pineko/evolve.py

```diff
@@ -99,7 +99,12 @@ def write_operator_card(pineappl_grid, default_card, card_path, tcard):
 
     """
     operators_card = copy.deepcopy(default_card)
-    x_grid, _pids, _mur2_grid, muf2_grid = pineappl_grid.axes()
+    # Note that this is enough but once we consider max_al different
+    # from 0, it will be better to use the actual order_mask
+    mock_order_mask = np.array([True for _ord in pineappl_grid.orders()])
+    evolve_info = pineappl_grid.raw.evolve_info(mock_order_mask)
+    x_grid = evolve_info.x1
+    muf2_grid = evolve_info.fac1
```

Comment:
Since the whole pattern is repeated twice in this file, and also somewhere else, you can create your `axes()` function:
```py
def axes(grid):
    mock_order_mask = np.array([True for _ord in grid.orders()])
    evolve_info = pineappl_grid.raw.evolve_info(mock_order_mask)
    return evolve_info.x1, evolve_info.fac1

x_grid, muf2_grid = axes(pineappl_grid)
```

Reply:
[5b3a12e](https://github.com/NNPDF/pineko/pull/75/commits/5b3a12e7e5f565ddcb5d4511a554868059912c47)